CONTRIBUTING.md

```diff
@@ -131,25 +74,26 @@ server {
 }
 ```
 
-#### Elastic
+Then ensure NGINX service is started/reloaded on your system.
+
+#### ElasticSearch
+
+Monocle relies on ElasticSearch as backend to store Changes and Change' events.
 
 ```ShellSession
 ./contrib/start-elk.sh 9200
 ```
 
-#### APIv1
+#### API
 
-```ShellSession
-./contrib/start-apiv1.sh 9878
-```
+To start the Monocle API then run the following commands.
 
-#### Web
+To reload modules when code is updated then type `:reload`.
 
-```ShellSession
-./contrib/start-web.sh
-```
+In addition you could run `ghcid -c 'cabal repl monocle'` in another terminal, then on every save in your IDE you'll see a list of the errors and warnings from ghc.
```

Comment:
Perhaps we don't need to bother with `ghcid`, I think the ghci usage is good enough, then interested users are probably aware of ghcid, or other editor integrations.

Reply:
I'll prefer to keep it if you don't mind to much.